lib/brains/management/commands/braintree_config.py

```diff
@@ -40,23 +48,23 @@ def product_exists(plans, external_id, amount):
     Check that the product exists in Braintree.
     """
     if external_id not in plans:
-        log.warning(
-            'The plan: {0} does not exist in Braintree. Plans cannot be '
-            'created from the API and must be created from the Braintree '
-            'console.'
-            .format(external_id))
-
-        raise CommandError('Missing product: {0}'.format(external_id))
+        raise BraintreePlanDoesNotExist(
```

Comment:
this is just for the config script (I doubt we'd need it anywhere else). It needed to raise an error so that the message could be displayed with more top-level data.